sql_datafreshness/sql_datafreshness.py

```diff
@@ -141,7 +144,7 @@ def read_opts_file(file_name):
 
     return opts, thresholds
 
-def produce_report(data, general_threshold, adhoc_thresholds, msg_from, msg_to=''):
+def produce_report(args, data, general_threshold, adhoc_thresholds, msg_from, msg_to=''):
```

Comment:
The function is getting variables got from the params. This new param is an object with everything and it breaks the style followed by the original function. Why don't you pass the boolean args.send?